src/plugins/dbusrecv/README.md

```diff
@@ -0,0 +1,45 @@
+- infos = Information about the dbus plugin is in keys below
+- infos/author = Markus Raab <elektra@libelektra.org>
```

Comment:
You can put your name here.

Is this copied or moved from dbus plugin?

Reply:
Sure :)